src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/ReportedAdjudicationController.kt

```diff
@@ -108,11 +108,10 @@ class ReportedAdjudicationController {
     @RequestParam(name = "status") status: Optional<ReportedAdjudicationStatus>,
     @PageableDefault(sort = ["dateTimeOfIncident"], direction = Sort.Direction.DESC, size = 20) pageable: Pageable
   ): Page<ReportedAdjudicationDto> {
-    val reportDate = startDate.orElse(LocalDate.now())
     return reportedAdjudicationService.getAllReportedAdjudications(
       agencyId,
-      reportDate,
-      endDate.orElse(reportDate),
+      startDate.orElse(LocalDate.now().minusDays(3)),
```

Comment:
Just an FYi (it's OK in this case because someone already did it) but we ought not add logic to the controller as it's harder to unit test

Reply:
true, but not sure this logic is specific to the service, just to the rest end point, ie its to handle the optionals in a clean way